backend/src/app.ts

```diff
@@ -139,13 +137,28 @@ app.get('/search', async (req, res) => {
     const apts: ApartmentWithId[] = aptDocs.map(
       (apt) => ({ id: apt.id, ...apt.data() } as ApartmentWithId)
     );
+    app.set('landlords', landlords);
+    app.set('apts', apts);
+
+    const v: any = {};
```

Comment:
The backend doesn't need to send any data back to the frontend in this case, so you actually don't even need `v` here. That also means you can change the response code to `204` which means `No Content` in API design: see this example [here](https://www.rockyourcode.com/express.js-send-responses/)

Reply:
Oops accidentally pressed approve instead of request changes